relayer/chains/cosmos/stride/module.go

```diff
@@ -0,0 +1,53 @@
+package stride
+
+import (
+	"encoding/json"
+	fmt "fmt"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+var (
+	_ module.AppModuleBasic = AppModuleBasic{}
+)
+
+// AppModuleBasic implements the AppModuleBasic interface
+type AppModuleBasic struct{}
+
+func (AppModuleBasic) Name() string {
+	return "interchainquery"
+}
+
+func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
+	fmt.Println("App module basic register legacy amino")
+	RegisterLegacyAminoCodec(cdc)
+}
+
+func (a AppModuleBasic) RegisterInterfaces(reg cdctypes.InterfaceRegistry) {
+	fmt.Println("App module basic register legacy amino")
```

Comment:
This `Println` call could be dropped now

Reply:
good catch, removed